ludwig/model_export/onnx_exporter.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
+
+import torch
+
+from ludwig.api import LudwigModel
+
+
+class OnnxExporter(ABC):
+    def __init__(self, model):
+        self.model = model
+
+    @abstractmethod
+    def forward(self, x):
+        return self.model({"image_path": x})
+
+    def export_classifier(self, model_path, export_path):
+        ludwig_model = LudwigModel.load(model_path)
+        model = OnnxExporter(ludwig_model.model)  # Wrap the model
+        model.eval()  # inference mode, is this needed.. I think onnx export does this for us
+
+        width = ludwig_model.config["input_features"][0]["preprocessing"]["width"]
+        height = ludwig_model.config["input_features"][0]["preprocessing"]["height"]
+        example_input = torch.randn(1, 3, width, height, requires_grad=True)
+
+        torch.onnx.export(
+            model,
+            example_input,
+            export_path,
+            opset_version=18,
```

Comment:
Should `opset_version` and other `onnx` args be an arg in `export_classifier`?

Maybe it can look like:
```
def export_classifier(self, model_path, export_path, onnx_export_args_override):
```

Reply:
added